invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/DocumentsSearch/DocumentsSearch.js

```diff
@@ -120,31 +121,32 @@ export class DocumentsSearch extends Component {
     return <div>{totalResults} results</div>;
   };
 
+  // FIXME: this overrides whatever configuration we are passing from invenio.
```

Comment:
Should this be fixed/removed?

Reply:
It was removed on the other PR that is coming after it :/